--- calligra-2.9.11/cmake/productsets/desktop.cmake~	2016-02-02 20:53:13.000000000 +0100
+++ calligra-2.9.11/cmake/productsets/desktop.cmake	2021-11-15 09:23:34.884800706 +0100
@@ -21,6 +21,4 @@
         APP_CONVERTER
         FILEMANAGER
         OKULAR
-        # docs
-        DOC
 )
--- calligra-2.9.11/CMakeLists.txt~	2021-11-15 09:31:59.000000000 +0100
+++ calligra-2.9.11/CMakeLists.txt	2021-11-15 09:38:58.617049671 +0100
@@ -1034,10 +1034,6 @@
   add_subdirectory(active)
 endif()
 
-if(SHOULD_BUILD_DOC)
-    add_subdirectory(doc)
-endif()
-
 if(SHOULD_BUILD_PART_QTQUICK)
     add_subdirectory(qtquick)
 endif()
